Cargo.toml

```diff
@@ -7,5 +7,6 @@ license = "Apache-2.0"
 
 [dependencies]
 config = "0.9.2"
+http = "0.1"
```

Comment:
You'd be better off using the Uri crate directly, as there is a utility crate that will handle downloads, and I don't want people to think they should just download using the `http` crate directly.

Reply:
Unfortunately there are problems with the Uri crate:
1. it doesn't support the Clone trait and 
2. the code isn't available 

I am looking for alternatives